site/templates/origin__3_harryette_mullen.php

```diff
@@ -1,7 +1,64 @@
-<?php snippet('header'); ?>
+<?php snippet('header', ['headerClass' => 'mullen']); ?>
+<div class="navigation__container">
+  <nav>
+    <ul class="navigation">
+      <li class="navigation__item highlight"><a href="<?= $site->url(); ?>"><?= $site->title(); ?></a></li>
+    </ul>
+  </nav>
+</div>
+
 <div class="layout-wrapper">
-  <div class="text">
-    <?= $page->main_content()->kt(); ?>
+
+
+    <div class="poem">
+    <?= $page->poem()->kt(); ?>
+    </div>
+
+
+    <div class="poem_citation">
```

Comment:
proper BEM naming should be `poem__citation` (note two underscores)

Reply:
should also have an addition class for text since you're using kirby text. `poem__citation text`